src/RA_PopupWindows.h

```diff
@@ -41,11 +42,11 @@ class PopupWindows
 };
 
 //	Exposed to DLL
-extern "C"
-{
-    API extern int _RA_UpdatePopups(ControllerInput* input, float fDTime, bool Full_Screen, bool Paused);
-    API extern int _RA_RenderPopups(HDC hDC, RECT* rcSize);
-}
+_EXTERN_C
+_NODISCARD API int _RA_UpdatePopups(_In_ const ControllerInput* const __restrict input, _In_ float fDTime,
+                                    _In_ bool Full_Screen, _In_ bool Paused);
```

Comment:
Does changing the const-ness affect the ability to resolve the method via GetProcAddress?
https://github.com/RetroAchievements/RAIntegration/blob/master/src/RA_Interface.cpp#L453

I assume the annotations do not.

Reply:
The annotations don't. I think it might affect GetProcAddress. I'll suppress the const rule for the overlay and popup exports.